yetanotherco/aligned_layer
# feat (batcher): verify gnark proofs

Batcher now verifiers different gnark proofs

# To test:

1. Start anvil, batcher, aggregator & operator
2. Turn on debug logs with `RUST_LOG=debug` environment variable
3. From a different terminal, run:
    - `make batcher_send_plonk_bn254_task`
    - `make batcher_send_plonk_bls12_381_task`
    - `make batcher_send_groth16_bn254_task`
4. See for each of those verification result is true on batcher
5. Batcher will not send batch to s3 & Ethereum unless the blockchain advances on block. There is two ways to do this:
- Start anvil with specified block time
- Send a transaction from another terminal
